### ISSUE TITLE
Add_tags cron throwing GEOS exception

### ISSUE THREAD

**Issue by GeoWill** [DemocracyClub/EveryElection#1326]
When the [add_tags cron](https://github.com/DemocracyClub/ee_deploy/blob/b3b08ba24048a4740aa3e7f916f61d6d6c006203/crontab.yml#L52) runs we get an error that looks like this:

```
Loading data...
...data loaded.
Reading data from NUTS_Level_1_(January_2018)_Boundaries
Setting tags: NUTS1 to {'value': 'North East (England)', 'key': 'UKC'}...
Sentry is attempting to send 1 pending error messages
Waiting up to 10 seconds
Press Ctrl-C to quit
Traceback (most recent call last):
  File "/var/www/every_election/env/lib/python3.6/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.InternalError_: GEOSPointOnSurface: TopologyException: Input geom 1 is invalid: Self-intersection at or near point -7.5572590721153539 49.767266675496913 at -7.5572590721153539 49.767266675496913
```

**Comment by GeoWill** [comment]
I thought it might be useful to explain how I fixed it, since it involved poking at the db rather than altering the code. 

The error points to some invalid geometries. This can be confirmed in the console by doing:

```
qs = OrganisationDivision.objects.filter(geography__geography__isvalid=False)
len(qs)
>>> 25
```

I then grabbed that list of `DivisionGeography` ids and used it to filter the `Organisations_DivisionGeography` layer in qgis. With a smallish number geoms it's actually pretty ok to just use QGIS as a client straight into the RDS instance. I then used the 'check validity' tool which outputs a 'invalid reason' point layer, taking me to where the issue is. I could then fix the issues using QGIS node edit tool.

I could've done things like `geom = geom.buffer(0)` or such, but given the number involved it felt worth going and having a check of each one. This was useful because there were various artifacts from the digitisation that just needed to be deleted. 
 
I also discovered that I'd imported one set of wards with the wrong srid. So to fix that I went to the `elections_election` table and deleted the relevant `division_geography_ids`. I then deleted the `division_geographies`, then re-imported the boundaries (this time with the `--srid=` flag). Finally did an:
 
```
UPDATE elections_election ee
SET ee.division_geography_id = dg.id
FROM organisations_divisiongeography dg
WHERE
   ee.division_id = dg.division_id and
   ee.organisation_id=409 and 
   ee.poll_open_date = '2022-05-05' and
   ee.group_type is null	   
```  

I then checked there weren't any more with: 

```
SELECT *
FROM
    organisations_divisiongeography dg join
    organisations_organisationdivision od
        on dg.division_id = od.id join
    organisations_organisationdivisionset ods
        on od.divisionset_id = ods.id join
    organisations_organisation oo
        on ods.organisation_id = oo.id join
    organisations_organisationgeography og
        on oo.id = og.organisation_id
where not st_intersects(dg.geography, og.geography)
```

This doesn't do anything to stop it happening again in the future, but having some sort of flag that makes us aware of invalid geometries is not bad thing.
